SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-localtest.md

```diff
@@ -1,15 +1,15 @@
 <%(TOC)%>
-# Multiserver Shooter Tutorial
+# Multiserver shooter tutorial
```

Comment:
```suggestion
# Multiserver zoning shooter tutorial
```
Show it's not the same as the multiserver offloading tutorial - which I will rename slightly in a different task and PR.